src/lib/Scenes/Home/Components/Sales/index.tsx

```diff
@@ -116,7 +116,7 @@ export default createPaginationContainer(
     `,
   },
   {
-    getConnectionFromProps: ({ viewer }) => viewer && viewer.sale_artworks,
+    getConnectionFromProps: ({ viewer }) => viewer && (viewer.sale_artworks as ConnectionData),
```

Comment:
In my branch this block is no longer in this file but I've made sure to update on my side

Reply:
👍